py/desitarget/cmx/cmx_cuts.py

```diff
@@ -446,12 +446,12 @@ def isSTD_calspec(ra=None, dec=None, cmxdir=None, matchrad=1.,
     cmxdir = _get_cmxdir(cmxdir)
     # ADM get the CALSPEC objects.
     cmxfile =  os.path.join(cmxdir,'calspec.fits')
-    cals = fitsio.read(cmxfile)
+    cals = io.read_external_file(cmxfile)
 
     # ADM match the calspec and sweeps objects.
     calmatch = np.zeros_like(primary, dtype='?')
     cobjs = SkyCoord(ra*u.degree, dec*u.degree)
-    ccals = SkyCoord(cals['RA']*u.degree, cals["Dec"]*u.degree)
+    ccals = SkyCoord(cals['RA']*u.degree, cals["DEC"]*u.degree)
 
```

Comment:
I didn't explicitly test this, but I think this will fail if the external RA, Dec columns already have the `unit` attribute (which may not be true for the *current* `calspec.fits` file, but may be true in the future and/or for other external catalogs). In `cuts._gal_coords` I did something like:
```
    import astropy.units as u
    from astropy.coordinates import SkyCoord

    if hasattr(ra, 'unit') and hasattr(dec, 'unit') and ra.unit is not None and dec.unit is not None:
        c = SkyCoord(ra.to(u.deg), dec.to(u.deg))
    else:
        c = SkyCoord(ra*u.deg, dec*u.deg)
```